### ISSUE TITLE
Fusionauth (open source alternative to auth0)

### ISSUE THREAD

**Issue by alessandrocaprarelli** [junaid33/opensource.builders#242]
It is an open source alternative to auth0

**Comment by jcklpe** [comment]
@alessandrocaprarelli Closed source should go in the title, with suggestions in the comments/descriptions. Thanks

**Comment by jcklpe** [comment]
Fusionauth was Alessandro's suggestion. 

I also know of keycloak and glu as possible alternatives.

**Comment by saz** [comment]
Isn't fusionauth free, but not open source?

**Comment by jcklpe** [comment]
@saz  I'm not familar with it but I think you're correct.

**Comment by alessandrocaprarelli** [comment]
Yes @saz, you are right. I thought it was open source and free but it turned out to be only free.

**Comment by leqlaz777** [comment]
WSO2 Identity Server may be?

**Comment by agustif** [comment]
I'm a small contributor of this auth library for TypeScript

accountsjs.com/

**Comment by coudot** [comment]
LemonLDAP::NG : Free WebSSO software providing CAS, SAML and OpenID Connect support.
https://lemonldap-ng.org/
https://gitlab.ow2.org/lemonldap-ng/lemonldap-ng
https://github.com/LemonLDAPNG/

**Comment by jcklpe** [comment]
https://goauthentik.io/
